lib/acl.js

```diff
@@ -510,7 +509,11 @@ function allow(mode, req, res, next) {
     var reqPath = res && res.locals && res.locals.path ? res.locals.path : req.path;
     var acl = reqToACL(req);
     acl.allow(mode, reqPath, function(err) {
-        next(err);
+        if (err) {
```

Comment:
I will have a look at this before the meeting, I could be wrong, but I have been reasoning about this, and I think yours is better design, since we always pass a next.